tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -161,7 +178,35 @@ local utils = import 'templates/utils.libsonnet';
     },
   },
   local hf_glue = common.PyTorchTest {
```

Comment:
Since this is the common ancestor of all of the tests, can you move it to the top of the file?

Reply:
Moved.